librz/include/rz_vector.h

```diff
@@ -89,6 +89,18 @@ static inline void *rz_vector_index_ptr(RzVector *vec, size_t index) {
 	return (char *)vec->a + vec->elem_size * index;
 }
 
+// returns a pointer to the head of the vector
+static inline void *rz_vector_head(RzVector *vec) {
+	rz_return_val_if_fail(vec, NULL);
+	return (char *)vec->a;
```

Comment:
Void *?

Reply:
I copied it from the rest of the vector code. Not sure why it has `(char*)` there.